<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBasse</class>
 <widget class="QMainWindow" name="DataBasse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1317</width>
    <height>622</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1317</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
margin: 0;
padding: 0;
font-family: sans-serif;
  background: #191919;
  color: white;
  text-transform: uppercase;
  font-weight: 500;}
#centralwidget{
  padding: 30px;
  background: #191919;
  color: white;
  text-transform: uppercase;
  font-weight: 500;}
#TitleTable{
color: white;
  text-transform: uppercase;
  font-weight: 500;
}
QPushButton{
  border:0;
  background: none;
  display: block;
  text-align: center;
  border: 2px solid #2ecc71;
  padding: 14px 40px;
  outline: none;
  color: white;
  border-radius: 24px;
  transition: 0.25s;
  cursor: pointer;
}
QPushButton:hover{
  background: #2ecc71;
}
QLineEdit{border:0;
  text-align: center;
  border: 2px solid #3498db;
  padding: 14px 10px;
   background: #191919;
color: white;
  border-radius: 24px;
  transition: 0.25s;}
QLineEdit:focus{
  width: 280px;
  border-color: #2ecc71;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="TitleTable">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>62</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Table Title</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnAdistation">
          <property name="minimumSize">
           <size>
            <width>153</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ADISTATION</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDiplome">
          <property name="minimumSize">
           <size>
            <width>153</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>DIPLOME</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAddStudent">
          <property name="minimumSize">
           <size>
            <width>153</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Add Student</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnprintpdf">
          <property name="minimumSize">
           <size>
            <width>153</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Print PDF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRefresh">
          <property name="minimumSize">
           <size>
            <width>153</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>598</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="inputSearch">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSearch">
          <property name="minimumSize">
           <size>
            <width>153</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Searsh</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1317</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
